product_internal_reference_generator/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Product Internal Reference Generator",
```

Comment:
**question**: should this `exclude` `product_variant_default_code`?

Reply:
Thanks for review! Yes these modules makes no sence to be intalled together.